pages/app-layout/with-drawers.page.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import {
+  AppLayout,
+  ContentLayout,
+  Header,
+  HelpPanel,
+  NonCancelableCustomEvent,
+  SpaceBetween,
+  SplitPanel,
+  Toggle,
+} from '~components';
+import appLayoutLabels from './utils/labels';
+import { Breadcrumbs, Containers } from './utils/content-blocks';
+
+export default function WithDrawers() {
+  const [activeDrawerId, setActiveDrawerId] = useState<string | null>(null);
+  const [hasDrawers, setHasDrawers] = useState(true);
+  const [hasTools, setHasTools] = useState(false);
+  const [isToolsOpen, setIsToolsOpen] = useState(false);
+
+  const [widths, setWidths] = useState<{ [id: string]: number }>({
+    security: 500,
+    'pro-help': 280,
+  });
+
+  const drawers = !hasDrawers
+    ? null
+    : {
+        drawers: {
+          ariaLabel: 'Drawers',
+          activeDrawerId: activeDrawerId,
+          onResize: (event: NonCancelableCustomEvent<{ size: number; id: string }>) => {
+            const obj = widths;
+            obj[event.detail.id] = event.detail.size;
+            setWidths(obj);
+            console.log(widths.security);
```

Comment:
Minor: no needed?

Reply:
Work in progress page as commented above: https://github.com/cloudscape-design/components/pull/971#discussion_r1170081360

I will keep it unchanged it to not introduce extra merge conflicts for others